scripts/limpar-prefixo-interno-fonte-dados.py

```
@@ -0,0 +1,164 @@
+        if st in (200, 204):
+            ok += 1
+        else:
+            print(f"  FALHOU {c['slug']}: {st}")
+    print(f"gravadas: {ok} de {len(planos)}")
+
+    restou = sum(1 for c in paginar("candidatos?select=fonte_dados")
+                 if any(isinstance(f, str) and f.lower().startswith(PREFIXO)
+                        for f in (c.get("fonte_dados") or [])))
+    print(f"readback: {restou} ficha(s) ainda com '{PREFIXO}' (esperado 0)")
+    if restou:
+        raise SystemExit("ABORTA: readback ainda encontra o prefixo")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)